packages/atomic/src/components/result-template-components/result-template-decorators.tsx

```diff
@@ -31,5 +31,18 @@ export function ResultContext() {
       }
       return connectedCallback && connectedCallback.call(this);
     };
+
+    component.render = function () {
+      if (this.error) {
+        this.host?.remove();
+        console.error(
+          'Result component is in error and has been removed from the DOM',
```

Comment:
We can resolve the name of the element from this.host

Reply:
Since I'm also logging `this.error` and `this`, as a dev, in the console, you will have access to the full name of the component.